### PR TITLE
bump up yui-relayer to v0.4.5 and implement `ChainInfo::Timestamp`

### PR DESCRIPTION
**Pull request by siburu**

### CODE REVIEW

File: pkg/relay/ethereum/chain.go [hunk 1]
```diff
@@ -94,6 +94,15 @@ func (c *Chain) LatestHeight() (ibcexported.Height, error) {
 	return clienttypes.NewHeight(0, bn), nil
 }
 
+func (c *Chain) Timestamp(height ibcexported.Height) (time.Time, error) {
+	ht := big.NewInt(int64(height.GetRevisionHeight()))
+	if header, err := c.client.HeaderByNumber(context.TODO(), ht); err != nil {
+		return time.Time{}, err
+	} else {
+		return time.Unix(int64(header.Time), 0), err
```
Comment (bluele) [inline pkg/relay/ethereum/chain.go:102]: nits: As elsewhere, I prefer to return `nil` explicitly instead of `err`

Comment (siburu) [reply pkg/relay/ethereum/chain.go:102]: Me too. In fact, this is a mistake. I will fix it immediately. Thanks.

Comment (siburu) [reply pkg/relay/ethereum/chain.go]: Fixed.

**Review by bluele — APPROVED** [review]
@siburu LGTM 👍

**Pull request merged**